demos/ns-inputter--select/index.html

```diff
@@ -0,0 +1,20 @@
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+</head>
+
+<body>
+  <main class="ndsn">
+    <ns-inputter validation="[&quot;isRequired&quot;]" heading="Select an option" helper="" name="select-an-option" value="">
+      <label slot="label" for="nsid-3y58v053ysr-input">Select an option</label>
+      <select name="select" id="nsid-3y58v053ysr-input">
```

Comment:
Suggest a friendly `for` and `id` value instead of an auto generated one.
```suggestion
      <label slot="label" for="select-input">Select an option</label>
      <select name="select" id="select-input">
```